Apps/Immunization/test/unit/Services.Test/ImmunizationServiceTests.cs

```diff
@@ -73,17 +73,16 @@ public async Task ShouldGetImmunizations(bool canAccessDataSource)
                     LoadState = new PhsaLoadState
                         { RefreshInProgress = false },
                     Result = new ImmunizationResponse(
-                        new List<ImmunizationViewResponse>
-                        {
+                        [
                             new()
                             {
                                 Id = Guid.NewGuid(),
                                 Name = "MockImmunization",
                                 OccurrenceDateTime = DateTime.SpecifyKind(DateTime.Now, DateTimeKind.Unspecified),
                                 SourceSystemId = "MockSourceID",
                             },
-                        },
-                        new List<ImmunizationRecommendationResponse>()),
+                        ],
+                        []),
```

Comment:
Nit: now that these lines are much shorter, they can probably all fit on a single line.

Reply:
Need to configure Rider.  Change next time.